app/Http/Controllers/SupplyController.php

```diff
@@ -82,11 +82,12 @@ public function find(Request $request)
         return $response;
     }
 
+    # we do it here because ORIGIN may be configured elsewhere with randomization of hostname
     public function findScript(Request $request)
     {
         $params = [json_encode($request->getSchemeAndHttpHost()), json_encode(config('app.aduser_endpoint'))];
 
-        $jsPath = config('app.env') == 'production' ? resource_path('assets/js/tmp-copy/find.x.js') : resource_path('assets/js/tmp-copy/find.min.js');
+        $jsPath = public_path('-/find.js');
 
         $response = new StreamedResponse();
         $response->setCallback(function () use ($jsPath, $request, $params) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "$request" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1481)